docs: update plugin documentation (#888)
jackyzha0/quartz: docs/plugins/NotFoundPage.md

---
title: NotFoundPage
tags:
  - plugin/emitter
---

> This plugin is designed to generate a custom 404 (Not Found) page for the website to handle broken or non-existent URLs.

Reviewer: ```suggestion
This plugin emits 404 pages for handling broken/non-existent URL.
```

Author: Done
